examples/demo/src/App.tsx

```diff
@@ -0,0 +1,126 @@
+import { HorizontalSlider } from '@algolia/ui-components-horizontal-slider-react';
+import React from 'react';
+
+import '@algolia/ui-components-horizontal-slider-theme';
+import './App.css';
+import './Hit.css';
+
+type ProductHit = {
+  objectID: string;
+  category: string;
+  imageLink: string;
+  name: string;
+  price: number;
+  _score: number;
+  reviewScore: number;
+  reviewCount: number;
+  url: string;
+};
+
+const items: ProductHit[] = [
```

Comment:
We can move these items to a separate file to see and edit the `App` file more easily.

Reply:
Done in https://github.com/algolia/recommend/pull/50/commits/f5438dd5a69aeeff9227ac8000a66c1cdd96b61f